app/src/main/java/lt/vilnius/tvarkau/entity/Problem.java

```diff
@@ -107,6 +110,8 @@ public String getId() {
         return id;
     }
 
+    public String getIdForVilniusAccount() { return idForVilniusAccount; }
```

Comment:
Recommend using Lombok's `@Data` annotation for the class, so there would be no need to write this boilerplate code for getters and setters.